src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.CommandWords.DELETE_MODEL;
+import static seedu.address.logic.commands.CommandWords.SESSION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ENDTIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MOD_CODE;
```

Comment:
Can you change PREFIX_MOD_CODE to PREFIX_MODULE as we had discussed last Monday?

Reply:
Okie will do this in a separate PR!